presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -294,7 +294,11 @@ protected boolean filterSchema(String schemaName)
                 }
                 if (columnMapping.isEmpty()) {
                     UnsupportedTypeHandling unsupportedTypeHandling = getUnsupportedTypeHandling(session);
-                    verify(unsupportedTypeHandling == IGNORE, "Unsupported type handling is set to %s, but toPrestoType() returned empty", unsupportedTypeHandling);
```

Comment:
I find the original to be more readable. Spreading a not-overly-long check onto 5 lines is excessive. Also, this makes the diff harder to read.

Reply:
it was over our overall line limit.
i can always push something slightly over the limit (nothing prevents me) in the interest of readability, but would that mean our limit is actually few characters higher than it is?

i don't wan to be overly principled about line length in general, but l feel breaking is justified here.